Source/GameBaseFramework/Public/GameFramework/Experiences/GBFExperienceDefinition.h

```diff
@@ -44,4 +56,7 @@ class GAMEBASEFRAMEWORK_API UGBFExperienceDefinition final : public UPrimaryData
     // List of additional action sets to compose into this experience
     UPROPERTY( EditDefaultsOnly, Category = Gameplay )
     TArray< UGBFExperienceActionSet * > ActionSets;
+
+    UPROPERTY( EditDefaultsOnly, Category = Gameplay )
+    TMap< FString, FGBFExperienceAdditionalActionSets > OptionToAdditionalActionSetsMap;

```

Comment:
I'd rather have a `TArray< FGBFExperienceAdditionalActionSets >` here, and move the FString in the structure